webapp/components/setMaxBalance.tsx

```diff
@@ -48,7 +48,8 @@ export const SetMaxEvmBalance = function ({
   } = useNativeTokenBalance(token.chainId)
 
   const { balance: walletTokenBalance } = useTokenBalance(
-    token,
+    token.chainId,
+    token.address,
     !isNativeToken(token),
```

Comment:
```suggestion
    !isNativeToken(token),
```

Same as the previous comments

Reply:
Done.